examples/forall-param-reductions.cpp

```diff
@@ -41,118 +41,124 @@ constexpr int HIP_BLOCK_SIZE = 256;
 constexpr int SYCL_BLOCK_SIZE = 256;
 #endif
 
-int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
+int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
```

Comment:
@johnbowen42 do you know why it made this change? In other places, '*' and '&', etc. were moved from the variable to the type specifier. Is it because of the macro?

Reply:
I don't.   I haven't tinkered with the `.clang-format` config file, but it's likely a result of those settings.  It's also `clang-format` version dependent.  Eg `clang-format-14` is different from `clang-format-16` even with the same config file.   I am working on enforcing the clang version in cmake right now.

Reply:
I changed the pointer/reference config, should be fixed now

Reply:
See[https://github.com/LLNL/RAJA/blob/25aee0350b4b9b39cf46bc09598ced0890f8c9b5/.clang-format#L11]( https://github.com/LLNL/RAJA/blob/25aee0350b4b9b39cf46bc09598ced0890f8c9b5/.clang-format#L11)